### PR TITLE
added mk6, mk7, and mk8

### PR DESCRIPTION
**Pull request by mherzl**
I could use mk8 for a project I'm working on.

### CODE REVIEW

File: fixed-vector/Data/Vector/Fixed/Internal.hs [hunk 1]
```diff
@@ -57,6 +57,18 @@ mk5 :: (Vector v a, Dim v ~ 5) => a -> a -> a -> a -> a -> v a
 mk5 a1 a2 a3 a4 a5 = vector $ C.mk5 a1 a2 a3 a4 a5
 {-# INLINE mk5 #-}
 
+mk6 :: (Vector v a, Dim v ~ 6) => a -> a -> a -> a -> a -> a -> v a
+mk6 a1 a2 a3 a4 a5 a6 = vector $ C.mk5 a1 a2 a3 a4 a5 a6
```
Comment (Shimuuar) [inline fixed-vector/Data/Vector/Fixed/Internal.hs]: you forgot to replace C.mk5 :). Otherwise LGTM

Comment (mherzl) [reply fixed-vector/Data/Vector/Fixed/Internal.hs]: Oops, good catch. I will update the PR to fix that.

**Comment by Shimuuar** [comment]
I also need to update  CI so I'll merge this PR tomorrow

File: fixed-vector/Data/Vector/Fixed/Internal.hs (hunk 1 above)
Comment (Shimuuar) [reply fixed-vector/Data/Vector/Fixed/Internal.hs]: Such mistakes are usually are obvious for everyone except author

**Comment by Shimuuar** [comment]
Rebased, squashed and merged. I'll make release tomorrow. It's too late to release software on unsuspecting world

**Pull request closed without merging**